Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods', '>= 1.10'
+gem 'cocoapods', '~> 1.12'
```

Comment:
I had to upgrade CocoaPods because I run into a CDN issue with that older version.

```
[!] CDN: trunk URL couldn't be downloaded: https://cdn.cocoapods.org/deprecated_podspecs.txt Response: URL using bad/illegal format or missing URL
```